Plugins/Bitbucket/BitbucketPullRequestForm.cs

```diff
@@ -163,7 +181,7 @@ private IEnumerable<BitbucketUser> GetBitbucketUsers()
             }
             return list;
         }
-
+        */
```

Comment:
please remove if not used

Reply:
This is what is a part of the "open space" in the Create Pull Request form.
Users/Reviewers are not implemented, there are several layers to that (default reviewers, required reviewers, many users) that are difficult to implement, so i disabled that completely (done in 2.51).

I will submit a separate PR for removal of this, so the cut is clean.